iavl/cli.py

```diff
@@ -349,28 +350,61 @@ def visualize(db, version, store=None, include_prev_version=False):
 )
 @click.option("--store", "-s")
 @click.option(
-    "--version",
-    help="the version to query, default to latest version if not provided",
+    "--start-version",
+    help="the version to start, default to 1",
+    default=1,
     type=click.INT,
 )
-def state_changes(db, version, store: Optional[str]):
+@click.option(
+    "--end-version",
+    help="the end version, default to latest version",
+    type=click.INT,
+)
+@click.option(
+    "--out-dir",
+    help="the output directory to save the data files",
+    type=click.Path(exists=True),
+    required=True,
+)
+def dump_changesets(db, start_version, end_version, store: Optional[str], out_dir: str):
```

Comment:
maybe can reuse extract changeset in [test_state_round_trip](https://github.com/crypto-com/python-iavl/blob/eb54992d97b6c534f7e7b64036562621edb81d12/iavl/cli.py#L420)

Reply:
done

Reply:
[high-end](https://github.com/crypto-com/python-iavl/blob/735a420ba3079f34161d705262aefe55c33bde25/iavl/cli.py#L431)